### PR TITLE
Improvements to WCS driver (follow-up for PR #521)

### PR DESCRIPTION
**Pull request by ajolma**
## What does this PR do?

Continue improving the metadata WCS datasets report.

## What are related issues/pull requests?

#515

## Tasklist

 - [x] Copy all relevant metadata from capabilities
 - [x] Copy all relevant metadata from coverage description
 - [ ] Add test case(s)
 - [x] Review
 - [x] Adjust for comments
 - [x] All CI builds and checks have passed

### CODE REVIEW

File: gdal/frmts/wcs/wcsutils.cpp [hunk 1]
```diff
@@ -645,14 +659,20 @@ CPLString GetKeywords(CPLXMLNode *root,
...
-                const char *epsg = "http://www.opengis.net/def/crs/EPSG/0/";
-                size_t pos = word.find(epsg);
-                if (pos == 0) {
-                    CPLString code = word.substr(strlen(epsg), std::string::npos);
-                    if (code.find_first_not_of(DIGITS) == std::string::npos) {
-                        epsg_codes.push_back(atoi(code));
-                        continue;
+                // crs, replace "http://www.opengis.net/def/crs/EPSG/0/"
+                // or "urn:ogc:def:crs:EPSG::" with EPSG:
+                const char *epsg[] = {
```
Comment (rouault) [inline gdal/frmts/wcs/wcsutils.cpp]: you can use ```cont char* const``` so that the array is entirely in read-only section
and same for spec array below

**Comment by rouault** [comment]
@ajolma Is there more work needed on this ? Asking since 2.3.0RC1 is planned today

**Comment by ajolma** [comment]
Some small things like tests :) but otherwise this could be good to go.

**Comment by rouault** [comment]
```
wcsutils.cpp:665:28: error: duplicate ‘const’
                 const char const *epsg[] = {
                            ^
wcsutils.cpp:683:28: error: duplicate ‘const’
                 const char const *spec[] = {
                            ^
```

**Comment by ajolma** [comment]
This could be merged. I fear the tests are not going to be done very soon.

**Comment by rouault** [comment]
I let you merge. You can use the drop-down box (the arrow at the right of) of the "Merge pull request" button to select "Squash and merge", so as to have a single commit

**Pull request merged**